scripts/exgdas_enkf_update.sh

```diff
@@ -362,6 +362,7 @@ cat > enkf.nml << EOFnml
    sattypes_oz(8) = 'mls30_aura',
    sattypes_oz(9) = 'ompsnp_npp',
    sattypes_oz(10) = 'ompstc8_npp',
+   sattypes_oz(11) = 'ompstc8_n20',
```

Comment:
I agree entirely regarding the inclusion of `iasi_metop-c` in `exgdas_enkf_update.sh`.  I asked about this in a separate email thread.   Thanks for catching `ompslp_npp`.  It makes sense to include this in `exgdas_enkf_update.sh` since it's been added to `exglobal_atmos_analysis.sh`.

I will update `exgdas_enkf_update.sh` with both changes.

Reply:
Done at 61e6458.